SIG Autoscaling 2021 Report first draft

gjtempleton opened the pull request:
<!--  Thanks for sending a pull request!  Here are some tips for you:
- If this is your first contribution, read our Getting Started guide https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
- If you are editing SIG information, please follow these instructions: https://git.k8s.io/community/generator
  You will need to follow these steps:
  1. Edit sigs.yaml with your change 
  2. Generate docs with `make generate`. To build docs for one sig, run `make WHAT=sig-apps generate`
-->

**Which issue(s) this PR fixes**:
<!--
*Automatically closes linked issue when PR is merged.
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
-->
Fixes #6366

gjtempleton commented:
Work we need to do before moving this out of draft (and to be discussed at SIG Meeting tomorrow):
* Update youtube playlist with meeting recordings
* Update list of KEPs progressed/being worked on
* scale-client Owners list
* Review of Github team membership

Question for you @tpepper, with the `Unique reviewers/approvers` for SIG-owned Packages, what level does this apply to? Just k/k code, or down to the level of Cloud-provider implementations in the Cluster Autoscaler?

gjtempleton commented:
/assign @tpepper 

I think this is in a state to be reviewed now

We're still in the process of following-up on two pieces of admin highlighted by this:
* Membership of the Github teams referred to in the SIG Readme
* Ownership of the Scale client, @mwielgus to follow this up

parispittman commented on sig-autoscaling/annual-report-2021.md:
```diff
@@ -4,15 +4,14 @@
+   - Continued improvements of VPA via community feedback
+   - Improved extensibility of Cluster Autoscaler
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
```
can you include links here to docs or open issues?

parispittman commented on sig-autoscaling/annual-report-2021.md:
```diff
@@ -23,60 +22,63 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
-   -
+   - Code contributors and reviewers
+   - Issue triage/response
+   - Whilst there are no areas with explicitly 2 or few owners, there are a number of areas where we have this few subject matter experts, acting as a significant bottleneck for significant changes
```
can you list some of those areas here for the readers?

parispittman commented on sig-autoscaling/annual-report-2021.md:
```diff
@@ -23,60 +22,63 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes - though the core approvers are almost all from a single company.
```
can you list who that is and the other companies that contribute?

parispittman commented on sig-autoscaling/annual-report-2021.md:
```diff
@@ -23,60 +22,63 @@ In future, this will be generated from kubernetes/enhancements kep.yaml files
 
-   -
-   -
+   - Increased support for some cloud provider implementations for the Cluster Autoscaler
```
can you include links here that are actionable for end users who are reading the report?

gjtempleton commented:
Thanks for the review @parispittman 

I believe I've addressed all the comments. Have pushed it up as a separate commit for ease of reviewing the changes, but happy to squash down once you're happy with it.

parispittman commented:
/lgtm
thanks @gjtempleton !
